docs/cql/ddl.rst

```diff
@@ -198,6 +200,34 @@ An example that excludes a datacenter while using ``replication_factor``::
     DESCRIBE KEYSPACE excalibur
         CREATE KEYSPACE excalibur WITH replication = {'class': 'NetworkTopologyStrategy', 'DC1': '3'} AND durable_writes = true;
 
+.. _initial-tablets:
+
+The experimental ``initial_tablets`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``initial_tablets`` option is used to specify how many tablets the table is split into.
+It is only valid when ``experimental_features: tablets`` is specified in ``scylla.yaml`` (which
+in turn requires ``consistent_cluster_management: true``. It must be a power of two.
+
+A good rule of thumb to calculate initial_tablets is to divide the expected total storage used
+by tables in this keyspace by (``replication_factor`` * 5GB). For example, if you expect a 30TB
```

Comment:
One could think that he should sum the storage of all tables, so maybe clarify that he should take the storage of the largest table. And clarify that the option controls initial tablets for every table created in the keyspace.

Reply:
yes; but we should eventually make it a per-table option.